blog/articles/orchestrating-cloud-events-with.md

```diff
@@ -0,0 +1,187 @@
+# Orchestrating Cloud Events (with Knative and Zeebe)
+
+A couple of weeks ago, I presented at the **Knative Meetup** ([Video](https://www.youtube.com/watch?v=vOSsHp-6gto), [Slides](https://www.slideshare.net/salaboy/orchestrating-cloud-events-knative-meetup-2020)) about how you can leverage the Cloud-Native workflow engine [Zeebe](http://zeebe.io) to understand, enhance and orchestrate your applications that are already using [Cloud Events](https://cloudevents.io).
+
+I wanted to expand a bit on how these tools can help you gain a deeper understanding of how your distributed applications are working.
+
+You can find the Demo application, installation instructions, and some videos of the application and the tools in action on [GitHub](https://github.com/salaboy/orchestrating-cloud-events/).
+
+The application that I built consists of four microservices that use [Cloud Events](https://cloudevents.io/) to communicate and perform the typical (happy path) flow of the application: **“Buy Concert Tickets”**. Each service runs as Knative Service using Knative Eventing for exchanging messages with other services.
+
+![Knative Application](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-service-knative.png?raw=true)
+
+The application heavily relies on Knative Eventing as all the services emit events to the broker and register [Knative Triggers (Subscriptions)](https://github.com/salaboy/customer-waiting-room-app/blob/master/charts/customer-waiting-room-app/templates/ktriggers.yaml#L1) to the events that they are interested in.
+
+The following events are being emitted by the application for every single user who wants to buy tickets on our website:
+
+![Application Events](https://github.com/salaboy/orchestrating-cloud-events/blob/master/imgs/tickets-events.png?raw=true)
+
+> Note that this Cloud-Native application uses [WebSockets](https://en.wikipedia.org/wiki/WebSocket#:~:text=WebSocket%20is%20a%20computer%20communications,being%20standardized%20by%20the%20W3C.), which adds an extra layer of complexity when you think about scaling it.
+
+Even if you have tracing (with something like [OpenTracing](https://opentracing.io) and a centralized logging system, understanding what is going on inside your event-driven applications is quite tricky. Think about it like this -- lets say we have only 10 customers trying to buy tickets at once, understanding where each customer is in the process, or where potential bottlenecks are appearing is challenging.
+
+Until this point, this is just a normal Knative application and, because you are using Knative Eventing which natively supports Cloud Events, you can integrate more tools to the ecosystem as you will see in the next section.
```

Comment:
```suggestion
Until this point, this is just a normal Knative application, since you are using Knative Eventing which natively supports Cloud Events, you can integrate more tools to the ecosystem as you will see in the next section.
```

Maybe "since" instead of and, because. It reads a little bit better, but I can also see where the "and" would read alright.

Reply:
@mpetason up to you.. really.. both work fine for me..